containers/h5ad-to-arrow/Dockerfile

```diff
@@ -9,6 +9,19 @@ RUN apt-get update &&\
 COPY requirements-freeze.txt .
 RUN pip install  -r ./requirements-freeze.txt
 
+# Download the GRCh38 Ensembl release 100 to convert gene IDs to gene names.
+# Need to use custom URLs with http:// rather than ftp:// for travis,
```

Comment:
Didn't know there was an FTP issue on travis... Is this what you mean? https://blog.travis-ci.com/2018-07-23-the-tale-of-ftp-at-travis-ci

Reply:
I believe so. For example this build errored with https://travis-ci.com/github/hubmapconsortium/portal-containers/builds/177964055

```
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: http://ftp.ensembl.org/pub/release-99/gtf/homo_sapiens/Homo_sapiens.GRCh38.cdna.all.fa.gz
```